app/forms/schools/year2020_form.rb

```diff
@@ -5,28 +5,18 @@ class Year2020Form
     include ActiveModel::Model
     include ActiveModel::Serialization
 
-    attr_accessor :school_id, :induction_programme_choice, :core_induction_programme_id, :full_name, :email, :participants
-
-    validates :induction_programme_choice, presence: true, on: :choose_induction_programme
+    attr_accessor :current_user, :school_id, :induction_programme_choice, :core_induction_programme_id, :full_name, :email, :participants
```

Comment:
Do we need to keep `induction_programme_choice` in attributes?

Reply:
good spot, that can be removed